app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".sample.nfc.NfcScanningActivity" />
```

Comment:
Shall we lock the orientation to `Portrait` to avoid handling the state?

Reply:
@lydiasama I'm thinking about adding ViewModel to the project. Most of Google's samples (if not all) use ViewModel, they also push for this MVVM architecture. I think it's a basic thing in Android now. So I think it's fine to add them to the project without too much boilerplate, e.g., `Input` and `Output` interfaces, `BaseViewModel`, etc. What say you?

If we add VM, it's okay to allow orientation changes, IMO.